packages/govuk-frontend/src/govuk/common/index.mjs

```diff
@@ -194,7 +194,6 @@ export function setFocus($element, options = {}) {
  * Some browsers will load and run our JavaScript but GOV.UK Frontend
  * won't be supported.
  *
- * @internal
  * @param {HTMLElement | null} [$scope] - HTML element `<body>` checked for browser support
```

Comment:
```suggestion
 * @param {HTMLElement | null} [$scope] - (internal) `<body>` HTML element checked for browser support
```

That parameter is only really used for testing, especially as [the snippet in our docs](https://frontend.design-system.service.gov.uk/import-javascript/#before-you-start) adds the class to the `<body>` element in any case. Best to not encourage people to play with it, especially as you might misinterpret that you can pass any element and know if GOV.UK Frontend is supported for it (which is not what would happen).

Just documenting it is lighter than wrapping the function before export in `all.mjs` 😊